Library/Homebrew/os/linux/kernel.rb

```diff
@@ -6,12 +6,7 @@ module Kernel
       module_function
 
       def version
-        return @version if @version
-
-        version = Utils.popen_read("uname", "-r").chomp
-        return Version::NULL unless version
-
-        @version = Version.new version
+        OS.kernel_version
```

Comment:
I'm not sure if we want to preserve this or not

Reply:
I think so unless you can verify it's unused in e.g. linuxbrew-core in which case: 🔥 it.

Reply:
Looks like this is only used in `brew` in `extend/os/linux/diagnostic` (including `below_minimum_version?`):

```console
% cd $(brew --repo)
% grep -r 'Linux::Kernel' *
Library/Homebrew/test/os/linux/diagnostic_spec.rb:    allow(OS::Linux::Kernel).to receive(:below_minimum_version?).and_return(true)
Library/Homebrew/extend/os/linux/diagnostic.rb:        return unless OS::Linux::Kernel.below_minimum_version?
Library/Homebrew/extend/os/linux/diagnostic.rb:          Your Linux kernel #{OS::Linux::Kernel.version} is too old.
Library/Homebrew/extend/os/linux/diagnostic.rb:          We only support kernel #{OS::Linux::Kernel.minimum_version} or later.
```

```console
% cd $(brew --repo homebrew/core)
% git remote get-url origin
https://github.com/Homebrew/linuxbrew-core
% grep 'Linux::Kernel' Formula/*
% grep 'Linux::' Formula/*
Formula/clang-format.rb:    if Formula["glibc"].any_version_installed? || OS::Linux::Glibc.system_version < Formula["glibc"].version
Formula/glibc.rb:    Glibc.version >= OS::Linux::Glibc.system_version
Formula/glibc.rb:      Your system's glibc version is #{OS::Linux::Glibc.system_version}, and Homebrew's glibc version is #{Glibc.version}.
Formula/llvm@7.rb:    if Formula["glibc"].any_version_installed? || OS::Linux::Glibc.system_version < Formula["glibc"].version
Formula/llvm@8.rb:    if Formula["glibc"].any_version_installed? || OS::Linux::Glibc.system_version < Formula["glibc"].version
Formula/llvm@9.rb:    if Formula["glibc"].any_version_installed? || OS::Linux::Glibc.system_version < Formula["glibc"].version
Formula/llvm.rb:    if Formula["glibc"].any_version_installed? || OS::Linux::Glibc.system_version < Formula["glibc"].version
```

Reply:
Cool! I think you can 🔥 then.

Reply:
So uh, what does 🔥 mean in this context? 😅

Reply:
Sorry: burn it down (i.e. delete it)